t/Util.pm

```diff
@@ -820,4 +824,65 @@ sub spawn_dns_server {
     return $server;
 }
 
+sub run_openssl_client {
+    my($opts) = @_;
+    my $port = $opts->{port} or croak("`port` is required!");
+    my $san = $opts->{san};
+    my $host = $opts->{host} // '127.0.0.1';
+    my $path = $opts->{path} // '/';
+    my $timeout = $opts->{timeout} // 2.0;
+    my $request = $opts->{request};
+    my $request_default = $opts->{request_default} // 0;
+    my $ossl_opts = $opts->{opts} // '';
+    my $ossl_cmd = $opts->{ossl_cmd} // 'openssl';
+
+    my $tempdir = tempdir();
```

Comment:
Are we using tempdir at all? If not, we can remove this. Otherwise, it might be a good idea to set `CLEANUP => 1` so that the temporary directory would be removed when the object is discarded.

Reply:
You're right.  We are not.  This has been removed in https://github.com/h2o/h2o/pull/2947/commits/507bd47dd6c7174525e77f698af658d4fe47d287.